bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -115,6 +115,25 @@ spec:
 
           bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
 
+          check_db_health() {
+            db=$1
+            if [[ ! -f ${db} ]]; then
+                  echo "${db} does not exist, skipping health check"
+                  return
+            fi
+            echo "Checking $db health"
+            serverid=$(ovsdb-tool show-log ${db} | grep "server_id" | sed 's/server_id: \(.*\)/\1/')
```

Comment:
can we quote `${db}` everywhere?

also might as well let sed do both the matching and replacing, and maybe we can use `[[:xdigit:]]\+` to match hex.

`ovsdb-tool show-log "${db}" | sed -ne '/server_id:/s/server_id: *\([[:xdigit:]]\+\)/\1/p'`

Reply:
Done, thanks for helping in making this better!